botutils/botconfig/config.go

```diff
@@ -141,7 +141,7 @@ func (r *configReader) updatePrivateKey(config *Config) error {
 func (r *configReader) updateIntegrationId(config *Config) error {
 	integrationIdStr := r.os.Getenv(IntegrationIdEnvVar)
 	if integrationIdStr != "" {
-		integrationId, err := strconv.Atoi(integrationIdStr)
+		integrationId, err := strconv.ParseInt(integrationIdStr, 10, 64)
```

Comment:
why this change?

Reply:
If you look at the assignment below (`config.Github.App.IntegrationID = integrationId`), `IntegrationID` is now an int64.